esp/applications/eclwatch/ldap_authorization_map.yaml

```diff
@@ -146,6 +146,11 @@ ldap:
          -  path: ConfigAccess
             resource: ConfigAccess
             description: Access to super computer environment
+      ws_resources:
+         Feature:
+         -  path: ResourcesAccess
```

Comment:
This works fine, but if I didn't know what it was and just read ResourcesAccess at face value I would be confused.  We could call it ResourceInfoAccess, ResourceServiceAccess, or ResourceQueryAccess,.. purely to make it more self describing.
If you do change it you would have to change where it's used below.. I think just in the ecm file.